libraries/AP_HAL_ChibiOS/hwdef/AET-H743-Basic/hwdef.dat

```diff
@@ -0,0 +1,217 @@
+PE13 SPI4_MISO SPI4
+PE14 SPI4_MOSI SPI4
+
+# two I2C bus
+I2C_ORDER I2C2 I2C1
+
+# I2C1
+PB6 I2C1_SCL I2C1
+PB7 I2C1_SDA I2C1
+
+# I2C2
+PB10 I2C2_SCL I2C2
+PB11 I2C2_SDA I2C2
+
+# ADC
+PC0 BATT_VOLTAGE_SENS ADC1 SCALE(1)
+PC1 BATT_CURRENT_SENS ADC1 SCALE(1)
+PA4 BATT2_VOLTAGE_SENS ADC1 SCALE(1)
+PA7 BATT2_CURRENT_SENS ADC1 SCALE(1)
+
+define HAL_BATT_MONITOR_DEFAULT 4
+define HAL_BATT_VOLT_PIN 10
+define HAL_BATT_CURR_PIN 11
+define HAL_BATT2_VOLT_PIN 18
+define HAL_BATT2_CURR_PIN 7
+define HAL_BATT_VOLT_SCALE 11.0
+define HAL_BATT_CURR_SCALE 40.0
+define HAL_BATT2_VOLT_SCALE 11.0
+
+PC4 PRESSURE_SENS ADC1 SCALE(2)
+define HAL_DEFAULT_AIRSPEED_PIN 4
+
+PC5 RSSI_ADC ADC1
+define BOARD_RSSI_ANA_PIN 8
+
+# LED
+# green LED1 marked as B/E
+# blue LED0 marked as ACT
+PE3 LED0 OUTPUT LOW GPIO(90) # blue
+PE4 LED1 OUTPUT LOW GPIO(91) # green
+define HAL_GPIO_A_LED_PIN 91
+define HAL_GPIO_B_LED_PIN 90
+
+define HAL_GPIO_LED_ON 1
+define AP_NOTIFY_GPIO_LED_2_ENABLED 1
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 OTG2 USART6 UART7 UART8
+
+# USART1
+PA10 USART1_RX USART1
+PA9  USART1_TX USART1
+
+# USART2
+PD5 USART2_TX USART2
+PD6 USART2_RX USART2
+
+# USART3
+PD9 USART3_RX USART3
+PD8 USART3_TX USART3
+
+# UART4
+PB9 UART4_TX UART4
+PB8 UART4_RX UART4
+
+# USART6
+PC7 TIM3_CH2 TIM3 RCININT PULLDOWN LOW
+PC6 USART6_TX USART6 NODMA
+
+# as an alternative config setup the RX6 pin as a uart. This allows
+# for bi-directional UART based receiver protocols such as FPort
+# without any extra hardware
+PC7 USART6_RX USART6 NODMA ALT(1)
+
+# UART7
+PE7 UART7_RX UART7
+PE8 UART7_TX UART7
+PE10 UART7_CTS UART7
+PE9  UART7_RTS UART7
+
+# UART8
+PE0 UART8_RX UART8
+PE1 UART8_TX UART8
+
+# Serial functions
+define DEFAULT_SERIAL1_BAUD 115
+define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_MAVLink2
+define DEFAULT_SERIAL2_BAUD 230
+define DEFAULT_SERIAL2_PROTOCOL SerialProtocol_GPS
+define DEFAULT_SERIAL3_BAUD 115
+define DEFAULT_SERIAL3_PROTOCOL SerialProtocol_MAVLink2
+define DEFAULT_SERIAL4_BAUD 230
+define DEFAULT_SERIAL4_PROTOCOL SerialProtocol_GPS
+define DEFAULT_SERIAL6_BAUD 115
+define DEFAULT_SERIAL6_PROTOCOL SerialProtocol_RCIN
+define DEFAULT_SERIAL7_BAUD 115
+define DEFAULT_SERIAL7_PROTOCOL SerialProtocol_MAVLink2
+define DEFAULT_SERIAL8_BAUD 115
+define DEFAULT_SERIAL8_PROTOCOL SerialProtocol_MAVLink2
+
+# CAN bus
+PD0 CAN1_RX CAN1
+PD1 CAN1_TX CAN1
+PD3 GPIO_CAN1_SILENT OUTPUT PUSHPULL SPEED_LOW LOW GPIO(70)
+
+# Motors
+PB0  TIM8_CH2N TIM8  PWM(1)  GPIO(50)
+PB1  TIM8_CH3N TIM8  PWM(2)  GPIO(51)
+PA0  TIM5_CH1  TIM5  PWM(3)  GPIO(52)
+PA1  TIM5_CH2  TIM5  PWM(4)  GPIO(53)
+PA2  TIM5_CH3  TIM5  PWM(5)  GPIO(54)
+PA3  TIM5_CH4  TIM5  PWM(6)  GPIO(55)
+PD12 TIM4_CH1 TIM4 PWM(7) GPIO(56)
+PD13 TIM4_CH2 TIM4 PWM(8) GPIO(57)
+PD14 TIM4_CH3 TIM4 PWM(9) GPIO(58)
+PD15 TIM4_CH4 TIM4 PWM(10) GPIO(59)
+PE5  TIM15_CH1 TIM15 PWM(11) GPIO(60)
+PE6  TIM15_CH2 TIM15 PWM(12) GPIO(61)
+PA8 TIM1_CH1 TIM1 PWM(13) GPIO(62) # for WS2812 LED
```

Comment:
you will need a defaults.parm file to enable the LED....ie SERVO12_FUNCTION 120, and NTF_LED_TYPES 257